src/android/java/io/jxcore/node/BtToRequestSocket.java

```diff
@@ -1,8 +1,6 @@
 package io.jxcore.node;
 
 import android.bluetooth.BluetoothSocket;
-import android.os.Handler;
-import android.util.Log;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.net.ServerSocket;
```

Comment:
mmSocket and readyCallback should both be final.

Also mStopped is set in lots of places but is never actually checked. What purpose does it serve?